verification/src/lib.rs

```diff
@@ -17,3 +18,4 @@ pub use http_server::{
     handlers::verification::{VerificationResponse, VerificationResult, VerificationStatus},
     run as run_http_server, AppRouter, Router,
 };
+pub use network::make_retrying_request;
```

Comment:
may be add a new line between mods and use?

Reply:
@leviska, sorry, didn't get it. where do you suggest adding new line? after 20th line?

Reply:
yes